### PR TITLE
Updated index.html file, added search option

### PR DESCRIPTION
**Pull request by OdedHellman**

### CODE REVIEW

File: superx/templates/index.html [hunk 1]
```diff
@@ -58,31 +66,20 @@
...
       <h1 class="display-3 text-center"><strong>SuperX</strong></h1>
-      <p >כאן יוכנס טקסט בהמשך</p>
-      <!-- <p><a class="btn btn-primary btn-lg" href="#" role="button">לבדיקת מחירים לחץ כאן!</a></p> -->
-      <div class="dropdown">
+      <p>השווה מחירים בעזרת כפתור החיפוש</p>
+      <form id="searchbutton" class="form-inline mt-2 mt-md-0">
+          <input class="form-control mr-sm-2 text-center" type="text" placeholder="הקלד מוצר לחיפוש" aria-label="Search">
+          <button class="btn btn-outline-success my-2 my-sm-0 text-center" type="submit">חפש</button>
+      </form>
+      <!-- <div class="dropdown">
```
Comment (simzacks) [inline superx/templates/index.html]: Why are you leaving this comment in here?

File: superx/templates/index.html [hunk 2]
```diff
@@ -92,14 +89,14 @@ <h1 class="display-3 text-center"><strong>SuperX</strong></h1>
...
-      </div>
+      </div> -->
     </div>
     </div>
 
 
   <div class="container">
     <!-- Example row of columns -->
-    <div class="row">
+    <!-- <div class="row">
```
Comment (simzacks) [inline superx/templates/index.html]: why are you leaving this comment in here?

File: superx/templates/index.html [hunk 3]
```diff
@@ -128,5 +124,5 @@ <h2>Test</h2>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
  <!-- Latest compiled and minified JavaScript -->
 <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>
-<script>$('.dropdown-toggle').dropdown();</script>
+<!-- <script>$('.dropdown-toggle').dropdown();</script> -->
```
Comment (simzacks) [inline superx/templates/index.html]: why leave this comment in here?

**Comment by OdedHellman** [comment]
All unnecessary comments were deleted.
I separated index.html into three dedicated files.

**Review by simzacks — APPROVED** [review]

**Pull request merged**